### PR TITLE
fixing StringIO and BytesIO for Python3

### PR DESCRIPTION
**Pull request by kmader**
urlopen returns string in python 2 and bytes in python 3 and so needs to be wrapped with BytesIO for python 3 and StringIO for python 2

### CODE REVIEW

File: examples/colab/tf_hub_delf_module.ipynb [hunk 1]
```diff
@@ -222,7 +222,8 @@
         "def download_and_resize_image(url, filename, new_width=256, new_height=256):\n",
         "  response = urlopen(url)\n",
         "  image_data = response.read()\n",
-        "  pil_image = Image.open(StringIO(image_data))\n",
+        "  image_data = BytesIO(image_data) if PY3 else StringIO(image_data)\n",
```
Comment (arnoegw) [inline examples/colab/tf_hub_delf_module.ipynb]: Is the case distinction really needed? six.BytesIO (just like six.StringIO) maps to StringIO.StringIO in Python 2, says https://pythonhosted.org/six/

Comment (kmader) [reply examples/colab/tf_hub_delf_module.ipynb]: ah I hadn't seen that I just knew it didn't work on Python 3 and didn't want to break the 2 support

**Comment by arnoegw** [comment]
Thanks! I'll shepherd this through our internal repo and back to the GitHub master branch.

**Comment by arnoegw** [comment]
This was now merged into the master branch as https://github.com/tensorflow/hub/commit/42d7bcea698b614dbc20cee52091108c0753d990

**Comment by kmader** [comment]
cool thanks!

**Pull request closed without merging**